.github/files/lint-project-structure.sh

```diff
@@ -216,6 +216,18 @@ for PROJECT in projects/*/*; do
 		fi
 	fi
 
+	# - if the project has any php files, a phan config should exist.
```

Comment:
Given that you've added the config/baseline to the generator skeletons, I guess people would only ever see this if they were creating a new project more manually. Though I wonder if you might reference the Static Analysis section you added to `docs/monorepo.md` just if someone runs into this error and is confused?

Reply:
Yeah, ideally people won't see this, but it's there in case someone tries to delete the `.phan` stuff, or adds a php file to something that didn't have any before, or something like that.

If you have suggestions for wording, I'd be happy to look at them. Otherwise I'll have to give it a little thought.

Reply:
Yeah I don't imagine it should come up often if at all. Suggestion below if you want to run with it or similar, but not a blocker.